workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-spring-boot/client-extension.yaml

```diff
@@ -22,6 +22,14 @@ liferay-sample-object-action-2:
     oAuth2ApplicationExternalReferenceCode: liferay-sample-oauth-application-user-agent
     resourcePath: /sample/object/action/2
     type: objectAction
+liferay-sample-object-action-3:
+    name: Liferay Sample Object Action 3
+    oAuth2ApplicationExternalReferenceCode: liferay-sample-oauth-application-user-agent
+    objectDefinitionNames:
+        - Object Definition 1 Name
+        - Object Definition 2 Name
```

Comment:
@murilostodolni objectDefinitionName cannot has spaces, [see](https://github.com/liferay/liferay-portal/blob/7ba4cf99abeced68976df84ee8551414e022b02f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java#L800-L813)

Reply:
rename to ObjectDefinition1Name and ObjectDefinition2Name